configure.ac

```diff
@@ -156,6 +156,14 @@ AC_CHECK_HEADERS(netdb.h netinet/in.h sys/socket.h)
 AC_CHECK_HEADERS(sys/param.h sys/select.h inttypes.h)
 AC_CHECK_HEADERS(arpa/inet.h winsock2.h)
 
+use_poll=yes
```

Comment:
Could you set poll() not to be used by default for now since it's a risky change?
I'd like to test it for a little bit before making it the default.

Use: AC_ARG_WITH() so that we can use:

```
$ ./configure --with-poll
```